apprentice/webdev/safety.md

```diff
@@ -5,7 +5,7 @@
 - Read the following points on web safety (taken from the [New York Public Library](https://www.nypl.org/help/about-nypl/legal-notices/internet-safety-tips)) Note: These points are focused at minors, but apply to anyone on the internet. 
     - Personal Information. Don’t give out personal information without your parents’ permission. This means you should not share your last name, home address, school name, or telephone number. Remember, just because someone asks for information about you does not mean you have to tell them anything about yourself.
     - Screen Name. When creating your screen name, do not include personal information like your last name or date of birth.
-    - Passwords. Don’t share your password with anyone but your parents. When you use a public computer make sure you logout of the accounts you’ve accessed before leaving the terminal.
+    - Passwords. Don’t share your password with anyone but your parents. When you use a public computer make sure you logout of the accounts you’ve accessed before leaving the terminal. [Do not save your passwords on public computers either.]
```

Comment:
I originally put in the brackets because it is not the original text from the website, but I'll go ahead and remove them.